lib/suse/connect/client.rb

```diff
@@ -240,17 +250,25 @@ def registered?
       # Refreshing the service instead to remove the repos of deregistered product.
       def remove_or_refresh_service(service)
         if service.name == 'SMT_DUMMY_NOREMOVE_SERVICE'
+          log.info '-> Refreshing service ...'
           Zypper.refresh_all_services
         else
+          log.info '-> Removing service from system ...'
           System.remove_service service
         end
       end
 
-      def print_success_message(product, action: 'Registered')
-        log.info "#{action} #{product.identifier} #{product.version} #{product.arch}"
+      def print_success_message(product, params)
```

Comment:
def print_success_message(product, action)
   if action == :register

Reply:
I think this is all that is required here, clearer than passing a hash with only entry.

Reply:
I couldn't reproduce the problem you described with the simple :register/:deregister parameter.

Reply:
Whuut...okay. I will check that again.